### ISSUE TITLE
Setting the default color incorrectly saves the rarity node

### ISSUE THREAD

**Issue by sarhatabaot** [sarhatabaot/TradingCards#90]
### Description

Setting the default color incorrectly saves the rarity node.

```
rarities:
    common:
        name: common
        display-name: '&7Common'
        default-color: '&7'
        buy-price: 100.0
        sell-price: 50.0
        rewards: [eco give %player% 100]
    uncommon:
        name: uncommon
        display-name: '&9Uncommon'
        default-color: '&9'
        rewards: ['']
    rare:
        name: rare
        display-name: '&eRare'
        default-color: '&e'
        rewards: ['']
    very-rare:
        name: very-rare
        display-name: '&6Very Rare'
        default-color: '&6'
        rewards: ['']
    legendary:
        name: legendary
        display-name: '&6Legendary'
        default-color: '&6'
        rewards: ['']
    treasure:
        name: treasure
        display-name: treasure
        default-color: ''
        rewards: []
treasure:
    name: treasure
    display-name: treasure
    default-color: '&6'
    rewards: []
```

### Steps to reproduce

1. Run `/cards edit rarity <rarity_id> DEFAULT_COLOR <color>`

### Expected Behaviour

The correct rarity node should update.

### Screenshots

![](https://puu.sh/IOphB/60056bd0d8.png)

### Server Software

Paper version git-Paper-216 (MC: 1.18.1)
TradingCards version 5.6.0-SNAPSHOT

### Minecraft Version

1.18.x

**Comment by sarhatabaot** [comment]
This also happens with display-name

**Comment by sarhatabaot** [comment]
Fixed in #93
